docs/reference/UDS Core/functional-layers.md

```diff
@@ -46,3 +46,41 @@ packages:
     repository: ghcr.io/defenseunicorns/packages/uds/core-backup-restore
     ref: 0.29.0-upstream
 ```
+
+## Layer Selection
+
+Layer selection will always be deployment-specific but below are guidelines for what layers to consider for your deployment.  The layers marked with a four pointed star (🟄) are those needed to follow the [Big Bang Conformant Stack](https://repo1.dso.mil/big-bang/product/bbtoc/-/blob/master/policy/conformance.md?ref_type=heads) though if you are not bound by that document, UDS Core Base is the only *technical* layer required to install most UDS Packages.
+
+| Layer | Selection Criteria |
+|-------|--------------------|
+| **UDS Add-ons*** |
+| UDS UI           | Provides a common operating picture for a Kubernetes cluster and UDS deployments <br/> *(install if you would like to have an easy-to-use window into your cluster/deployments)* |
+| UDS Registry     | Provides a storage location for UDS components and mission applications <br/> *(install if you would like to be able to easily store and view the software available in your environment)* |
+| UDS Remote Agent | Allows for more advanced remote cluster management / deployment <br/> *(install if you would like to manage UDS deployments from more advanced clients than UDS CLI)* |
```

Comment:
My leaning is to include this and the prerequisites _after_ the core layers, maybe even as separate tables - but open to other opinions.

Reply:
One additional reason to split these into separate tables - the 4 back to back note blocks could be split up and placed after each separate table. Then we wouldn't need any symbols (*,†,††,‡) except for the two core notes - and could potentially use one of those unused symbols to denote BBCS requirements.

My suggestion would be ordering as prereqs, then core layers, then addons - each as separate tables with their individual notes underneath.